cypress/e2e/models/migration/applicationinventory/application.ts

```diff
@@ -897,4 +897,24 @@ export class Application {
         Issues.openSingleApplication(this.name);
         cy.get(commonView.appTable).should("not.contain.text", appIssues);
     }
+    verifyAssessmentTakeButtonEnabled(): void {
+        //first check if we are the assessment actions page
+        cy.get("section.pf-v5-c-page__main-section.pf-m-light h1").then((h1) => {
+            if (!h1.text().includes("Assessment Actions")) {
+                Application.open();
+                this.clickAssessButton();
+            }
+        });
+        cy.contains("button", "Take", { timeout: 30 * SEC }).should(
+            "not.have.attr",
+            "aria-disabled",
+            "true"
+        );
+    }
+
+    discardAssessment(): void {
+        Application.open();
+        this.clickAssessButton();
+        cy.get(".pf-v5-c-table__tr.actions-row").find("button.pf-v5-c-button.pf-m-plain").click();
```

Comment:
Let's add some locators to assessment view.

Reply:
done